schema/encoding/jsonschema/benchmark_complex_test.go

```diff
@@ -0,0 +1,221 @@
+package jsonschema_test
+
+import (
+	"bytes"
+	"github.com/rs/rest-layer/schema"
+	"github.com/rs/rest-layer/schema/encoding/jsonschema"
+	"math"
+	"testing"
+	"time"
+)
+
+// Optional schema field
+func Optional(s schema.Field) schema.Field {
```

Comment:
Even test code should be clean, perhaps even more clean than the code itself. Tests also serve as documentation you know.

I understand that you copy-paste, but it would be cleaner to leave required = false as the default and remove the Optional method.